.gitignore

```diff
@@ -37,13 +68,9 @@ coverage.xml
 node_modules/
 bower_components/
 .grunt/
+vendor/
+static/vendor/
 src/vendor/
-src/static/fonts/
```

Comment:
Yeah, I saw that too. I think what happened was we stopped installing from bower to `/src`, but never updated the tasks to match. That said, we could really use a clean up of the tasks and should probably port them to Gulp so we're consistent with other projects. What say you?